<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>958</width>
    <height>280</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>931</width>
      <height>161</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="page">
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>32</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Registrars</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>60</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Subacute Beds</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>90</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Junior Residents</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>110</y>
        <width>231</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>View Results</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="minorOpBedsSpinBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>90</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="juniorResidentsSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>90</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>120</y>
        <width>81</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Interns</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>0</y>
        <width>201</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Resuscitation Beds</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="durationSpinBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>120</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="internsSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>30</y>
        <width>141</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Acute Beds</string>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>0</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Consultants</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>110</y>
        <width>151</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Simulation Time</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="registrarsSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>60</y>
        <width>171</width>
        <height>16</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Senior Residents</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="consultantsSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QPushButton" name="StartButton">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>10</y>
        <width>161</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Start</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="acuteBedsSpinBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>30</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>90</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 12pt &quot;Copperplate Gothic Light&quot;;</string>
      </property>
      <property name="text">
       <string>Minor Op Beds</string>
      </property>
     </widget>
     <widget class="QSpinBox" name="seniorResidentsSpinBox">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>161</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="resuscitationBedsSpinBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>0</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="subacuteBedsSpinBox">
      <property name="geometry">
       <rect>
        <x>790</x>
        <y>60</y>
        <width>141</width>
        <height>22</height>
       </rect>
      </property>
      <property name="maximum">
       <number>999999999</number>
      </property>
     </widget>
     <widget class="QTextEdit" name="textEdit">
      <property name="geometry">
       <rect>
        <x>350</x>
        <y>70</y>
        <width>231</width>
        <height>31</height>
       </rect>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Segoe UI'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_2">
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Patients(Treated+Those in beds):</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Total no of Patients waiting in Queue:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category 1 Patients in Queue:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category 2 Patients in Queue:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category 3 Patients in Queue:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category 4 Patients in Queue:</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Category 5 Patients in Queue:</string>
       </property>
      </item>
     </widget>
     <widget class="QListWidget" name="listWidget_2">
      <property name="geometry">
       <rect>
        <x>655</x>
        <y>10</y>
        <width>271</width>
        <height>151</height>
       </rect>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>282</x>
        <y>20</y>
        <width>371</width>
        <height>29</height>
       </rect>
      </property>
      <property name="text">
       <string>Go Back</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QCheckBox" name="CriticalEmergency">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>150</y>
      <width>331</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Sitka Small</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="autoFillBackground">
     <bool>true</bool>
    </property>
    <property name="text">
     <string>Critical Emergency</string>
    </property>
    <property name="tristate">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>958</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHospitalManagementSimulator">
    <property name="title">
     <string>HospitalManagementSimulator</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuMade_by_M_Meesam_DawoodAliKhan_FaqihAliZamin">
    <property name="title">
     <string>Made by: M.Meesam,DawoodAliKhan,FaqihAliZamin</string>
    </property>
   </widget>
   <addaction name="menuHospitalManagementSimulator"/>
   <addaction name="menuMade_by_M_Meesam_DawoodAliKhan_FaqihAliZamin"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
